README.md

```diff
@@ -1,6 +1,13 @@
-# Project Name
+# Using Terraform to deploy and configure Azure Front Door with an Azure App Service
 
-This project provides a sample terraform script for provisioning a WAF enabled frontdoor with backend pools set with an existing Azure App service, routing rules with  caching config exposed on a private vnet
+This project end to end terraform scripts for provisioning a WAF enabled Azure Front Door with backend pools set with an existing Azure App service, routing rules with caching config.
```

Comment:
err… apologies, the wording here is incomplete!! Should be: "This project provides end to end…"

Reply:
DOne! I had end to end wordings before and then removed them :) and have it back :)